routes/rendering-info/html-static.js

```diff
@@ -74,7 +74,11 @@ module.exports = {
           name: styleHashMap.default
         }
       ],
-      sophieModules: [],
+      sophieModules: [
+        {
+          name: "sophie-legend@1"
```

Comment:
This should be removed here and added in Q-server-nzz config here: https://github.com/nzzdev/Q-server-nzz/blob/dev/config/nzz/tools/production.js#L361 (and for all envs and targets) in the target specific variant (sophie-nzzas-legend, sophie-regio-legend), otherwise we load the nzz.ch styles for all targets.